cfme/tests/infrastructure/test_providers.py

```diff
@@ -117,16 +118,12 @@ def test_ip_required_validation():
 
 
 @pytest.mark.tier(3)
-@pytest.mark.xfail(message='http://cfme-tests.readthedocs.org/guides/gotchas.html#'
-    'selenium-is-not-clicking-on-the-element-it-says-it-is')
-def test_name_max_character_validation():
+def test_name_max_character_validation(provider, request):
     """Test to validate max character for name field"""
-    prov = VMwareProvider(
-        name=fauxfactory.gen_alphanumeric(256),
-        hostname=fauxfactory.gen_alphanumeric(5),
-        ip_address='10.10.10.12')
-    prov.create()
-    prov.delete(cancel=False)
+    provider.name = fauxfactory.gen_alphanumeric(255)
```

Comment:
@psav We can't go with "any" credentials because adding a provider without successful credentials validation no longer works. The `Add` button will remain disabled.

@otsuman Like Pete said, we cannot parametrize this per each provider. We have to run this test against a single provider but we don't need to care which one. So... we have 2 choices here:
A) We need to get a random provider and set it up ourselves with a crazy long name (this would require splitting https://github.com/ManageIQ/integration_tests/blob/master/utils/providers.py#L121 into `get_a_provider` and `setup_a_provider`)
or
B) use existing `setup_a_provider` to setup a random provider, then edit it with a crazy long name, check that it works and make sure there is a finalizer that will delete it afterwards or at least change the name back to the original one.